code/game/world.dm

```diff
@@ -240,7 +245,7 @@ GLOBAL_LIST_EMPTY(world_topic_handlers)
 	return s
 
 /world/proc/SetupLogs()
-	GLOB.log_directory = "data/logs/[time2text(world.realtime, "YYYY/MM-Month/DD-Day")]"
+	GLOB.log_directory = "data/logs/[time2text(world.realtime, "YYYY/MM-Month/DD-Day")]/round-[GLOB.round_id]"
```

Comment:
This feels like it would break when round_id is not set (because we have db-less setup for example.
I don't think it's worth fixing it here, instead make GLOB.round_id default to something sane (so it can be used in other places and not deal with `null` handling).

Reply:
![image](https://user-images.githubusercontent.com/25063394/102549175-977ff080-40b3-11eb-82f2-f4f6a3fed8d2.png)